app/models/manageiq/providers/openstack/inventory/collector/infra_manager.rb

```diff
@@ -1,3 +1,9 @@
 class ManageIQ::Providers::Openstack::Inventory::Collector::InfraManager < ManageIQ::Providers::Openstack::Inventory::Collector
   include ManageIQ::Providers::Openstack::Inventory::Collector::HelperMethods
+
+  def images
+    return [] unless image_service
+    return @images if @images.any?
+    @images = uniques(image_service.handled_list(:images))
```

Comment:
why not the more standard `@images ||= uniques(image_service.handled_list(:images))` ?

Reply:
Because `@images` is initialized with `[]` in the collector initializer so `||=` wouldn't actually fetch anything